src/pages/users/widgets/UsersTable.vue

```diff
@@ -0,0 +1,92 @@
+<script setup lang="ts">
+  import { defineVaDataTableColumns } from 'vuestic-ui'
+  import { User, UserRole } from '../types'
+  import { useUsersStore } from '../../../stores/users'
+  import UserAvatar from './UserAvatar.vue'
+
+  const columns = defineVaDataTableColumns([
+    { label: 'Full Name', key: 'fullname', sortable: true },
+    { label: 'Email', key: 'email', sortable: true },
+    { label: 'Username', key: 'username', sortable: true },
+    { label: 'Role', key: 'role', sortable: true },
+    { label: 'Projects', key: 'projects', sortable: true },
+    { label: ' ', key: 'actions', align: 'right' },
+  ])
+
+  const users = useUsersStore()
+
+  const roleColors: Record<UserRole, string> = {
+    admin: 'danger',
+    user: 'background-element',
+    owner: 'warning',
+  }
+
+  defineEmits<{
+    (event: 'edit-user', user: User): void
+    (event: 'delete-user', user: User): void
```

Comment:
We need a notification on user delete. (Are you sure you want to delete this user?)

Then we also need confirmation message for all actions: User has been successfully deleted/added/updated

![image](https://github.com/epicmaxco/vuestic-admin/assets/5394573/505e0538-9965-4709-aa7c-32dc7591007b)